<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_part</class>
 <widget class="QDialog" name="user_part">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Res/img/Logo.png</normaloff>:/Res/img/Logo.png</iconset>
  </property>
  <widget class="QLabel" name="lbl_BG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 234, 235);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_border">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>12</y>
     <width>1250</width>
     <height>690</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid rgb(47, 60, 126)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_resizer">
   <property name="geometry">
    <rect>
     <x>1255</x>
     <y>315</y>
     <width>15</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>230</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(47, 60, 126);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_search">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>45</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:4px solid rgb(47, 60, 126);
color: rgb(47, 0, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic_5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>370</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(47, 60, 126);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tbl_search">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>250</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(47, 60, 126);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic_4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>230</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(47, 60, 126);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="GP_SearchMethod">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>250</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(47, 60, 126);</string>
   </property>
   <property name="title">
    <string>Search by :</string>
   </property>
   <widget class="QRadioButton" name="radio_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="redio_Category">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="pic_1">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>90</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(47, 60, 126);



</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_description">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>500</y>
     <width>661</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(47, 60, 126);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic_0">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 5px solid rgb(47, 60, 126);


</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_3">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>370</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(47, 60, 126);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_search">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(47, 60, 126);
</string>
   </property>
   <property name="text">
    <string>Search :</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic_6">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(47, 60, 126);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_line">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>15</y>
     <width>5</width>
     <height>690</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid rgb(47, 60, 126);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pic_profile">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>90</y>
     <width>120</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(47, 60, 126);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addtocart">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>45</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 60, 126);
color: rgb(251, 234, 235);</string>
   </property>
   <property name="text">
    <string>+ &amp;Add to cart</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_avalablity">
   <property name="geometry">
    <rect>
     <x>749</x>
     <y>30</y>
     <width>171</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 154, 6);
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Not Available</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cart">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 60, 126);
color: rgb(251, 234, 235);</string>
   </property>
   <property name="text">
    <string>&amp;Cart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_logoff">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>660</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 60, 126);
color: rgb(251, 234, 235);</string>
   </property>
   <property name="text">
    <string>&amp;Log off</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ediprofile">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>130</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 60, 126);
color: rgb(251, 234, 235);</string>
   </property>
   <property name="text">
    <string>&amp;Edit Profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cart_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>170</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 60, 126);
color: rgb(251, 234, 235);</string>
   </property>
   <property name="text">
    <string>&amp;Payments</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_firstname">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(47, 60, 126);</string>
   </property>
   <property name="text">
    <string>Firstname :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_history">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>210</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 60, 126);
color: rgb(251, 234, 235);</string>
   </property>
   <property name="text">
    <string>&amp;Shop History</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_lastname">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>350</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(47, 60, 126);</string>
   </property>
   <property name="text">
    <string>Lastname :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_email">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>450</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(47, 60, 126);</string>
   </property>
   <property name="text">
    <string>Email :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_wallet">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>540</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(47, 60, 126);</string>
   </property>
   <property name="text">
    <string>Wallet :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_username">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>30</y>
     <width>250</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(47, 60, 126);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_firstname">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>300</y>
     <width>230</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(47, 60, 126);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_lastname">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>390</y>
     <width>230</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(47, 60, 126);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_email">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>480</y>
     <width>230</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(47, 60, 126);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_wallet">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>570</y>
     <width>230</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(47, 60, 126);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lbl_BG</zorder>
  <zorder>lbl_line</zorder>
  <zorder>lbl_border</zorder>
  <zorder>btn_resizer</zorder>
  <zorder>pic_2</zorder>
  <zorder>txt_search</zorder>
  <zorder>pic_5</zorder>
  <zorder>tbl_search</zorder>
  <zorder>pic_4</zorder>
  <zorder>GP_SearchMethod</zorder>
  <zorder>pic_1</zorder>
  <zorder>txt_description</zorder>
  <zorder>pic_0</zorder>
  <zorder>pic_3</zorder>
  <zorder>lbl_search</zorder>
  <zorder>pic_6</zorder>
  <zorder>pic_profile</zorder>
  <zorder>btn_addtocart</zorder>
  <zorder>lbl_avalablity</zorder>
  <zorder>btn_Cart</zorder>
  <zorder>btn_logoff</zorder>
  <zorder>btn_ediprofile</zorder>
  <zorder>btn_Cart_2</zorder>
  <zorder>lbl_firstname</zorder>
  <zorder>btn_history</zorder>
  <zorder>lbl_lastname</zorder>
  <zorder>lbl_email</zorder>
  <zorder>lbl_wallet</zorder>
  <zorder>txt_username</zorder>
  <zorder>txt_firstname</zorder>
  <zorder>txt_lastname</zorder>
  <zorder>txt_email</zorder>
  <zorder>txt_wallet</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
